examples/manifold/plot_manifold_sphere.py

```diff
@@ -103,7 +103,7 @@
 print("%s: %.2g sec" % ('ISO', t1 - t0))
 
 ax = fig.add_subplot(257)
```

Comment:
this is not actually deprecated, only the 0 meaning 10. I'm happy to replace all of them if you like. Personally I favor ``plt.subplots`` ;)